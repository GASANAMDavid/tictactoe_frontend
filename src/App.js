import React from 'react';

function App() {
  const [header, setHeader] = React.useState('Welcome to TicTacToe Game')
  return (
    <div className='container'>
      <h3 className='header'>{header}</h3>
    </div>
  );
}

export default App;
