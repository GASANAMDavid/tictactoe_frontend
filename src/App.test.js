import React from "react";
import { shallow } from "enzyme";
import App from "./App";
import toJson from "enzyme-to-json"

describe('App', () => {
  let app
  beforeEach(() => app = shallow(<App/>))
  it('renders correctly', () => {
    expect(toJson(app)).toMatchSnapshot()
  });

  it('has a header', () => {
    expect(app.find('.header').exists()).toBe(true)
  });

  
})

