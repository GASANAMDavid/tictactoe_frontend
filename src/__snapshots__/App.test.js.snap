// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="header"
  >
    Welcome to TicTacToe Game
  </div>
</div>
`;
